source/Octopus.Tentacle/Configuration/ITentacleConfiguration.cs

```diff
@@ -129,6 +129,8 @@ public interface IWritableTentacleConfiguration : ITentacleConfiguration
         /// </summary>
         void ResetTrustedOctopusServers();
 
+        bool SetTrustedOctopusServers(IEnumerable<OctopusServerConfiguration>? newServers);
```

Comment:
Is it worth adding a brief docstring here to say that this will overwrite the trusted servers with the new list?

Reply:
Good call, will do